config.json

```diff
@@ -1,6 +1,8 @@
 {
-    "deep_filter_path": "MediaProcessor/res/deep-filter-0.5.6-x86_64-unknown-linux-musl",
+    "deep_filter_path_unix": "MediaProcessor/res/deep-filter-0.5.6-x86_64-pc-windows-msvc.exe",
```

Comment:
Here you have the unix and windows paths inversed and the Windows path has the wrong slashes. Corrected file:

```json
{
    "deep_filter_path_unix": "MediaProcessor/res/deep-filter-0.5.6-x86_64-unknown-linux-musl",
    "deep_filter_path_windows": "MediaProcessor\\res\\deep-filter-0.5.6-x86_64-pc-windows-msvc.exe",
    "downloads_dir": "downloads",
    "ffmpeg_path_unix": "/usr/bin/ffmpeg",
    "ffmpeg_path_windows": "C:\\ffmpeg\\bin\\ffmpeg",
    "upload_folder": "uploads"
}
```

ffmpeg path will be different for you